lua/nvim-treesitter/parsers.lua

```diff
@@ -28,13 +28,25 @@ M.configs = setmetatable({}, {
   end
 })
 
+--- @return string[]
+local function get_parser_names()
+  local path = vim.fs.joinpath(vim.fn.fnamemodify(debug.getinfo(1, 'S').source:sub(2), ':p:h'), 'parsers')
```

Comment:
`get_package_path` could be moved here instead (since `install` imports `parsers` anyway?)